scripts/README.rst

```diff
@@ -1,68 +1,31 @@
-Feature Toggle Reporter:
-------------------------
+Feature Toggle Reporter
+-----------------------
 
-We make extensive use of a variety of feature toggles at edX. However, it has
-the downside of making the state of a deployed application somewhat opaque.
-This tool anaylzes data from the codebases of deployed applications, as well
-as their databases, to provide a summary of the feature toggles in use in a
-given deployment. For more information, please see:
+We make extensive use of a variety of feature toggles at edX. However, it has the downside of making the state of a deployed application somewhat opaque. This tool analyzes data from the codebases of deployed applications, as well as their databases, to provide a summary of the feature toggles in use in a given deployment. For more information, please see:
 
 https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html
 
-Prerequisites:
---------------
-
-In order to create a feature toggle report for a given deployment, you need
-two types of data: feature toggle annotation data and feature toggle data from
-an application's database.
+In order to create a feature toggle report for a given deployment, you need two types of data: feature toggle annotation data and feature toggle state data from an application's database.
 
 Feature Toggle Annotation Data
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use instructions found in `get feature toggles annotation data document <https://edx-toggles.readthedocs.io/en/latest/how_to/documenting_new_feature_toggles.html>`__ to collect annotations data.
+Any toggles that are annotated according to the instructions in
+`get feature toggles annotation data document <docs/how_to/documenting_new_feature_toggles.html>`_
+will be included in the report.
```

Comment:
```suggestion
Any toggles that are annotated according to the instructions in `get feature toggles annotation data document <docs/how_to/documenting_new_feature_toggles.html>`_ will be included in the report.
```

Reply:
Hah! Thanks. It's very hard for me to spot these.